Sources/DatadogObjc/Datadog+objc.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import Datadog
+
+@objcMembers
+public class DDAppContext: NSObject {
+    internal let sdkAppContext: Datadog.AppContext
+
+    // MARK: - Public
+
+    public init(mainBundle: Bundle) {
+        self.sdkAppContext = Datadog.AppContext(mainBundle: mainBundle)
+    }
+
+    override public init() {
+        self.sdkAppContext = Datadog.AppContext()
+    }
+}
+
+@objcMembers
+public class DDDatadog: NSObject {
+    // MARK: - Public
+
+    public static func initialize(appContext: DDAppContext, configuration: DDConfiguration) {
+        Datadog.initialize(
+            appContext: appContext.sdkAppContext,
+            configuration: configuration.sdkConfiguration
+        )
+    }
+
+    public static func setVerbosityLevel(_ verbosityLevel: DDSDKVerbosityLevel) {
+        switch verbosityLevel {
+        case .debug: Datadog.verbosityLevel = .debug
```

Comment:
Yes, so basically doing
```swift
@objc enum Something {
   case a
   case b
   case c
}
```
doesn't expose it to Objective-C 😞. This is probably because enums in Swift are way more powerful (nested types, indirect enums, computed propersies, ...) and this doesn't have corresponding construct in Objc.

Reply:
the only way to expose an enum to objective-c is to make it raw representatble by an `Int`

```swift
@objc enum Something: Int {
   case a
   case b
   case c
}
```

Reply:
Ah, true @iGranDav , this is right. In fact, I do expose enums to Objc:
```swift
@objc
public enum DDSDKVerbosityLevel: Int {
    case none
    case debug
    case info
    case notice
    case warn
    case error
    case critical
}
```

I just gave a wrong reason for the mapping @mariusc83 commented on. The reason for another enum is that `Datadog.verbosityLevel` is the optional in Swift SDK:
```swift
public static var verbosityLevel: LogLevel? = nil
```
but this was impossible to model in Objective-C (no support for optional enums in method signature), so I supported this as:
```swift
public static func setVerbosityLevel(_ verbosityLevel: DDSDKVerbosityLevel) {
}

public static func verbosityLevel() -> DDSDKVerbosityLevel {
}
```
by adding `.none` case to `DDSDKVerbosityLevel` enum. Dunno if there's a better way, @iGranDav ?

Reply:
Your `nil` enum value is implicit in objective-c since basically any value can be `nil`.  Having the `.none` option is indeed a more explicit way to do so.

Usually it's the one equals to `0`